docs/pages/server-access/getting-started.mdx

```diff
@@ -0,0 +1,384 @@
+
+   iii. Make sure ports: `9090`, `443`, `3023-3025`, and `3080` are all open.
+
+2. Create two new instances of your desired Linux distribution (such as CentOS, Debian, Ubuntu).
+
+   <Admonition type="tip" title="Tip">
+     Multiple Teleport Nodes can be run on the same Linux instance. This Guide details how to launch and connect two Nodes on different devices.
+   </Admonition>
+
+   i. Add an `A` record in your domain registrar entry for each Linux instance. 
+
+   ii. Map the first IP address to the `auth` subdomain and the second IP address to the `ssh` subdomain.  
+
+   iii. Create the necessary SSH keys and connect from your local machine to each using the following `ssh` command format:
+
+   - `aws % ssh -i "example.pem" ubuntu@ecX-X-XX-XXX-XX.us-north-2.compute.amazonaws.com`  
+  
+   Most cloud providers require SSH to initially access, configure, and provision your cloud instances. We'll configure and launch our instances then demonstrate how to use the `tsh` tool and Teleport in SSH mode thereafter.
+       
+3. Install Teleport on each instance.
+
+   ```bash
+   # Install Teleport
+   curl https://deb.releases.teleport.dev/teleport-pubkey.asc | sudo apt-key add -
+   sudo add-apt-repository 'deb https://deb.releases.teleport.dev/ stable main'
+   sudo apt-get update
+   sudo apt-get install teleport
+   ```
+
+4. Configure Teleport to use ACME and to automatically acquire a TLS certificate:
+
+   <Admonition type="tip" title="Tip">
+    Teleport will now automatically acquire an ACME-compliant TLS certificate by default. While you can use a third-party ACME client (like Let's Encrypt certbot), the configuration commands below will do the same.
+   </Admonition>
+
+   ```bash
+   # Configure Teleport
+   sudo teleport configure \
+     --acme --acme-email=your_email@example.com \
+     --cluster-name=auth.example.com \
+     -o file
+   ```
+
+5. Launch Teleport on `auth.example.com`: `sudo teleport start`. 
+
+## Step 1b. Add a Node to the cluster
+
+<video autoplay loop muted playsinline>
+  <source src="https://goteleport.com/teleport/videos/server-access/addnode.mp4" type="video/mp4"/>
+  <source src="https://goteleport.com/teleport/videos/server-access/addnode.webm" type="video/webm"/>
+Your browser does not support the video tag.
+</video>
+
+1. Now, create a third terminal and connect to `auth.example.com`:
+
+   i. Create a user to access the `auth.example.com` Web UI: 
+
+   - `sudo tctl users add auth-admin --roles=editor,access,admin --logins=root,ubuntu,ec2-user`  
+
+   ii. This will generate an initial login link where you can set a password and set up Two-Factor Authentication for `auth-admin`.
+
+   iii. Create a *join token* to link `ssh.example.com` with the `auth.example.com` cluster:
+
+   - `sudo tctl tokens add --type=node`  
+   - "Node" mode specifies that the Teleport Node that will join will act as an SSH server.  
+
+   iv. You should see:
+
+   ```bash
+   # Join cluster
```

Comment:
You can point the joined node to `tele.example.com:3080` it will be easier to setup as well. This will spin the node in IOT mode. The node will establish the tunnel to the proxy and then proxy will use the tunnel.